radixdlt/babylon-wallet-android
# [ABW-1672] Add info popup

## Description
This PR adds info buttons and info dialogs. It also:
- adds a separate composable for warnings: `WarningText`
- adds a separate composable for info buttons: `InfoButton`
- added `InfoDialog` composable screen that can be triggers from several places in the wallet (similar to asset dialogs)
- several UI fixes based on Zeplin

## How to test

1. Open Wallet and navigate to the screens that have a info button
2. click on this and click also on the other links inside the dialog
3. There are two types of links: web links that open the browser and the `glossaryAnchor`s that update the content of the info dialog: See the video


## Video

https://github.com/user-attachments/assets/1d7b1ab0-78c1-43c8-99a0-de5973f92ec5

## PR submission checklist
- [x] I have tested info buttons
